pandas/core/groupby/ops.py

```diff
@@ -415,7 +416,15 @@ def wrapper(*args, **kwargs):
 
         return func
 
-    def _cython_operation(self, kind, values, how, axis, min_count=-1, **kwargs):
+    def _cython_operation(
+        self, kind: str, values, how: str, axis, min_count: int = -1, **kwargs
+    ) -> Tuple[np.ndarray, Optional[List[str]]]:
+        """
+        Returns the values of a cython operation as a Tuple of [data, names].
+
+        Names is only useful when dealing with 2D results, like ohlc
```

Comment:
we should make a 1d and 2d version of this and just branch on the calling function to avoid a bunch of special case branches, prob post this PR

Reply:
Yea same comment on ohlc. When the dust settles and we get block management out of GroupBy I think I'll just rip that out of these methods so we can simplify further